azurecompute/src/main/java/org/jclouds/azurecompute/binders/DeploymentParamsToXML.java

```diff
@@ -50,21 +50,11 @@
                     .e("AdminPassword").t(params.password()).up()
                     .e("ResetPasswordOnFirstLogon").t("false").up()
                     .e("EnableAutomaticUpdate").t("false").up()
-                    .e("DomainJoin")
-                    .e("Credentials")
-                    .e("Domain").t(params.name()).up()
-                    .e("Username").t(params.username()).up()
-                    .e("Password").t(params.password()).up()
-                    .up() // Credentials
-                    .e("JoinDomain").t(params.name()).up()
-                    .up() // Domain Join
-                    .e("StoredCertificateSettings").up()
                     .e("WinRM")
                     .e("Listeners")
                     .e("Listener")
-                    .e("Protocol").t("http").up().up().up().up()
+                    .e("Protocol").t("Https").up().up().up().up()
                     .e("AdminUsername").t(params.username()).up()
-                    .e("AdminPassword").t(params.username()).up()
```

Comment:
what's the default `AdminPassword` ? Can you make add something on the README, maybe?
Do you think we never need to override that default value?